bin/release.sh

```diff
@@ -35,7 +35,7 @@ BROWSER_PACKAGE_CHANGED=$(npx lerna changed --parseable | grep -c packages/platf
 if [ -z "${RETRY_PUBLISH:-}" ]; then
   case $VERSION in
     "prerelease" | "prepatch" | "preminor" | "premajor")
-      npx lerna version "$VERSION"  --no-push --no-private --dist-tag next
+      npx lerna version "$VERSION"  --no-push --no-private
       ;;
```

Comment:
this is now a duplicate command to the other branch, so we can drop the branching logic here - otherwise all looks good!